src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.cs

```diff
@@ -0,0 +1,710 @@
+          Children(1): IOperation:  (OperationKind.None) (Syntax: 'default:')
+        IBranchStatement (BranchKind.Break) (OperationKind.BranchStatement) (Syntax: 'break;')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<DefaultSwitchLabelSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact, WorkItem(8884, "https://github.com/dotnet/roslyn/issues/8884")]
+        public void ParameterReference_UserDefinedLogicalConditionalOperator()
+        {
+            string source = @"
+class A<T>
+{
+    public static bool operator true(A<T> o) { return true; }
+    public static bool operator false(A<T> o) { return false; }
+}
+class B : A<object>
+{
+    public static B operator &(B x, B y) { return x; }
+}
+class C : B
+{
+    public static C operator |(C x, C y) { return x; }
+}
+class P
+{
+    static void M(C x, C y)
+    {
+        if (/*<bind>*/x && y/*</bind>*/)
+        {
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IOperation:  (OperationKind.None) (Syntax: 'x && y')
+  Children(2): IConversionExpression (ConversionKind.Cast, Implicit) (OperationKind.ConversionExpression, Type: B) (Syntax: 'x')
+      IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: C) (Syntax: 'x')
+    IConversionExpression (ConversionKind.Cast, Implicit) (OperationKind.ConversionExpression, Type: B) (Syntax: 'y')
+      IParameterReferenceExpression: y (OperationKind.ParameterReferenceExpression, Type: C) (Syntax: 'y')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<BinaryExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact, WorkItem(8884, "https://github.com/dotnet/roslyn/issues/8884")]
+        public void ParameterReference_NoPiaObjectCreation()
+        {
+            var sources0 = @"
+using System;
+using System.Runtime.InteropServices;
+
+[assembly: ImportedFromTypeLib(""_.dll"")]
+[assembly: Guid(""f9c2d51d-4f44-45f0-9eda-c9d599b58257"")]
+[ComImport()]
+[Guid(""f9c2d51d-4f44-45f0-9eda-c9d599b58277"")]
+[CoClass(typeof(C))]
+public interface I
+        {
+            int P { get; set; }
+        }
+[Guid(""f9c2d51d-4f44-45f0-9eda-c9d599b58278"")]
+public class C
+{
+    public C(object o)
+    {
+    }
+}
+";
+            var sources1 = @"
+struct S
+{
+	public I F(object x)
+	{
+		return /*<bind>*/new I(x)/*</bind>*/;
+    }
+}
+";
+            var compilation0 = CreateStandardCompilation(sources0);
+            compilation0.VerifyDiagnostics();
+
+            var compilation1 = CreateStandardCompilation(
+                sources1,
+                references: new[] { MscorlibRef, SystemRef, compilation0.EmitToImageReference(embedInteropTypes: true) });
+
+            string expectedOperationTree = @"
+IOperation:  (OperationKind.None) (Syntax: 'new I(x)')
```

Comment:
Yes, but the generated bound node doesn't seem to have it. I will file an issue.